### ISSUE TITLE
E1.31 Channel Inputs screen shows Interface dropdown for Channel Outputs

### ISSUE THREAD

**Issue by MarkAAmber** [FalconChristmas/fpp#449]
E1.31 Channel Inputs screen shows the Interface dropdown from the Channel Outputs screen.  This is confusing.  If you change it on this screen, thinking you are setting the interface the packets come in, you end up changing it for the Outputs.

Probably should be removed from the Inputs screen completely as I don't believe there is any reason to specify which interface the packets come in.  It should receive them on any interface and only need to be specified on the Output screen.

Likewise for the Input Universe Type.  I don't believe there is any reason to specify whether the received packet is Multicast or Unicast.  It should be received either way.

**Comment by dkulp** [comment]
Removed the dropdown for the interface.  You're right, that's pointless (and was configuring outputs anyway).   I did adjust the code to bind multicast to ALL ipv4 interfaces as it was just using eth0 and wlan0 so it wouldn't have worked for eth1 or or other wlan#'s.   Now fixed.

If multicast is selected (the default), it will listen to unicast as well (same socket/port#).   However, if unicast is selected, it will not send out the IGMP subscriptions and thus would not receive the multicast packets.  There IS a reason to allow configuring this.   At startup, we have to send out a IGMP for EACH universe on EACH network adapter.  Thus, if you have a lot of universes needed, startup will be noticeably slower.   Also, some routers/access points don't have the capacity to handle subscriptions on a BUNCH of multicast addresses and thus, the flood of IGMP's from one host may cause issues.  By allowing the selection of unicast, we can startup faster as well as workaround issues with cheaper routers and such.
